app/policies/evss_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+EVSSPolicy = Struct.new(:user, :evss) do
+  def access?
```

Comment:
it would be interesting if we could make these into verbs instead of nouns:

"access?" - not a verb
"read?" - verb
"write?" - verb
"eligible?" - not a verb

etc... and possibly have an initializer or something that forbids someone from creating a random verb (for the sake of consistency). Is there any way to limit what you can specify as these methods on the policy?

Reply:
I'm worried that "access" alone might be too generic. Does it imply GET, CREATE, and UPDATE? It would be nice if we could be more explicit about the specific controller action somehow.